app/controllers/media_objects_controller.rb

```diff
@@ -429,9 +431,11 @@ def update_status
             end
         end
       end
+      message = "#{success_count} #{'media object'.pluralize(success_count)} successfully #{status}ed."
+      message += "These objects were not #{status}ed:</br> #{ errors.join('<br/> ') }" if errors.count > 0
+    else
+      message = "Status must be one of 'publish' or 'unpublish'"
```

Comment:
Should this be I18n'd?

Reply:
Yes and no. Here's the "no" rationale: nothing in this app is I18n'd, we are unlikely to need to translate anything, and this code will likely end up in the trash heap when there is either a Jupiter integration or we move to the Hyrax version of the app.

But yes, in theory this should be I18n'd -- I'm just not sure working on a translation that will never be used is a good use of time at this point.

Reply:
ERA-AV/Avalon needs a lot of more work in other areas as well if we need to bring the app to be I18n'd. For now I'll open a new ticket for i18n but kick that can into the future.